src/frontends/tensorflow_common/src/helper_transforms/const_to_result_remover.cpp

```diff
@@ -2,24 +2,32 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include "helper_transforms/unsupported_const_to_result_remover.hpp"
+#include "helper_transforms/const_to_result_remover.hpp"
 
 #include "helper_ops/unsupported_constant.hpp"
+#include "openvino/opsets/opset10.hpp"
 
 using namespace std;
+using namespace ov::opset10;
 
 namespace ov {
 namespace frontend {
 namespace tensorflow {
 namespace pass {
 
-bool UnsupportedConstToResultRemover::run_on_model(const std::shared_ptr<ov::Model>& m) {
+bool ConstToResultRemover::run_on_model(const std::shared_ptr<ov::Model>& m) {
     ResultVector results_to_remove;
     // look for isolated UnsupportedConst->Result sub-graphs to remove
+    // also, find isolated Constant->Result sub-graphs to remove
     for (const auto& result : m->get_results()) {
         auto unsupported_const = as_type_ptr<UnsupportedConstant>(result->get_input_node_shared_ptr(0));
         if (unsupported_const && unsupported_const->output(0).get_target_inputs().size() == 1) {
             results_to_remove.push_back(result);
+            continue;
+        }
+        auto const_node = as_type_ptr<Constant>(result->get_input_node_shared_ptr(0));
+        if (const_node && const_node->output(0).get_target_inputs().size() == 1) {
```

Comment:
`const_node->output(0).get_target_inputs().size() == 1` I guess this check is unnecessary.
Constant will remain in the graph if we delete useless result here
![image](https://user-images.githubusercontent.com/21181928/219088828-f56a4c71-ebe7-43ff-81b7-0cdc9d9c9b11.png)

Reply:
@itikhono, optimized